robottelo/cli/base.py

```diff
@@ -1,28 +1,27 @@
 # -*- encoding: utf-8 -*-
 """Generic base class for cli hammer commands."""
 import logging
-import re
 
 from robottelo import ssh
 from robottelo.cli import hammer
 from robottelo.config import conf
 
 
-# Task status message have two formats:
-#   * "Task b18d3363-f4b8-44eb-871c-760e51444d22 success: 1.0/1, 100%,
-#     elapsed: 00:00:02\n"
-#   * "Task 6ba9d82a-ea55-4934-8aab-0e057102ee11 running: 0.005/1, 0%, 0.0/s,
-#     elapsed: 00:00:02, ETA: 00:06:55\n"
-TASK_STATUS_REGEX = re.compile(
-    r'Task [\w-]+ \w+: [\d./]+, \d+%,( [\d./s]+,)? elapsed: [\d:]+'
-    '(, ETA: [\d:]+)?\n\n?'
-)
-
-
 class CLIError(Exception):
     """Indicates that a CLI command could not be run."""
 
 
+class CLICommandExecutionError(Exception):
+    """Indicates that a CLI command returned an error after execution."""
```

Comment:
Also, can you make the docstring a little more precise? What's the difference between a command returning an error, versus one returning an error "after execution"? What constitutes an error? Just a status code, or does the presence of contents in stderr also warrant raising this exception? I'd be very happy to see a clear definition of when and why this exception is raised.